src/lib/wallet-connection.ts

```diff
@@ -42,7 +42,7 @@ const walletConnect2EthereumProviderOptions: Parameters<(typeof EthereumProvider
       '--wcm-accent-color': colors.colorpurple,
       '--wcm-background-color': colors.colorpurple,
       '--wcm-z-index': '10000',
-      '--wcm-font-family': 'Relative Pro',
+      '--wcm-font-family': 'Courier, monospace',
```

Comment:
Any reason we aren't using Satoshi here? Correct me if I'm wrong, but I don't think it is mean't to be mono.

Reply:
oh.. derp, you are right. not sure why I thought it was mono